python/ray/dashboard/subprocesses/handle.py

```diff
@@ -204,206 +190,121 @@ async def _do_periodic_health_check(self):
         Every 1s, do a health check. If the module is unhealthy:
         1. log the exception
         2. log the last N lines of the log file
-        3. fail all active requests
-        4. restart the module
+        3. restart the module
         """
         while True:
             try:
                 await self._do_once_health_check()
-            except Exception as e:
+            except Exception:
                 filename = module_logging_filename(
                     self.module_cls.__name__, self.config.logging_filename
                 )
+                if filename is None:
+                    filename = "stderr"
```

Comment:
why this?

Reply:
I missed these lines when splitting this PR into 2 PRs. Will remove this.